loops/fpc/code.pas

```diff
@@ -4,28 +4,20 @@
   SysUtils;
 
 var
-  u, r, i, j: Int32;
-  a: array[0..9999] of Int32;
+  u: UInt32;
+  a: array[0..9999] of UInt32;
```

Comment:
Interesting! Does using a unit give better performance than a signed iint?

Reply:
Surprisingly, yes, it's 40% more efficient.